src/wizard/__tests__/common.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { WizardProps } from '../interfaces';
+
+export const DEFAULT_I18N_SETS = [
```

Comment:
Can we replace the same settings in wizard.test.tsx with the common one?

Reply:
Done.